### PR TITLE
fix(components/indicators): adopt `@skyux/icons@5.3.1`

### PR DESCRIPTION
**Pull request by Blackbaud-SteveBrush**

### CODE REVIEW

File: libs/components/indicators/src/lib/modules/icon/icon.module.ts [hunk 1]
```diff
@@ -1,14 +1,13 @@
...
 import { NgModule } from '@angular/core';
-import { SkyThemeIconManifestModule } from '@skyux/theme';
 
 import { SkyIconClassListPipe } from './icon-class-list.pipe';
 import { SkyIconStackComponent } from './icon-stack.component';
 import { SkyIconComponent } from './icon.component';
 
 @NgModule({
   declarations: [SkyIconClassListPipe, SkyIconComponent, SkyIconStackComponent],
-  imports: [CommonModule, SkyThemeIconManifestModule],
```
Comment (Blackbaud-SteveBrush) [inline libs/components/indicators/src/lib/modules/icon/icon.module.ts:11]: This is an empty, deprecated module. No longer needed.

File: libs/components/packages/src/schematics/ng-add/ng-add.schematic.spec.ts [hunk 2]
```diff
@@ -88,14 +88,4 @@ describe('ng-add.schematic', () => {
       '@skyux/packages/polyfills',
     ]);
   });
-
-  it('should modify tsconfig.json', async () => {
```
Comment (Blackbaud-SteveBrush) [inline libs/components/packages/src/schematics/ng-add/ng-add.schematic.spec.ts:92]: There's no need for this schematic anymore.

File: libs/components/theme/src/index.ts [hunk 3]
```diff
@@ -1,6 +1,3 @@
-export * from './lib/icons/icon-manifest';
-export * from './lib/icons/icon-manifest-glyph';
-export * from './lib/icons/icon-manifest.module';
```
Comment (Blackbaud-SteveBrush) [inline libs/components/theme/src/index.ts:3]: All of these are internal, now exported by `@skyux/icons`.

File: tsconfig.base.json [hunk 4]
```diff
@@ -14,8 +14,8 @@
     "experimentalDecorators": true,
     "importHelpers": true,
     "target": "ES2022",
-    "module": "es2020",
-    "lib": ["es2020", "dom"],
+    "module": "ES2022",
+    "lib": ["es2022", "dom"],
```
Comment (Blackbaud-SteveBrush) [inline tsconfig.base.json:18]: These were missed in the recent Nx migration.

File: apps/e2e/ag-grid-storybook/tsconfig.json [hunk 5]
```diff
@@ -1,5 +1,8 @@
 {
   "extends": "../../../tsconfig.base.json",
+  "compilerOptions": {
+    "resolveJsonModule": true
```
Comment (Blackbaud-SteveBrush) [inline apps/e2e/ag-grid-storybook/tsconfig.json:4]: This is needed because this app imports from `@blackbaud/skyux-design-tokens/json/design-tokens.json`.

**Comment by blackbaud-sky-build-user** [comment]
[Storybook preview](https://blackbaud.github.io/skyux-pr-preview/1133/storybook/)

Component Storybooks:

- [a11y-storybook](https://blackbaud.github.io/skyux-pr-preview/1133/storybooks/a11y-storybook/)
- [ag-grid-storybook](https://blackbaud.github.io/skyux-pr-preview/1133/storybooks/ag-grid-storybook/)
- [core-storybook](https://blackbaud.github.io/skyux-pr-preview/1133/storybooks/core-storybook/)
- [indicators-storybook](https://blackbaud.github.io/skyux-pr-preview/1133/storybooks/indicators-storybook/)
- [layout-storybook](https://blackbaud.github.io/skyux-pr-preview/1133/storybooks/layout-storybook/)
- [modals-storybook](https://blackbaud.github.io/skyux-pr-preview/1133/storybooks/modals-storybook/)
- [pages-storybook](https://blackbaud.github.io/skyux-pr-preview/1133/storybooks/pages-storybook/)
- [popovers-storybook](https://blackbaud.github.io/skyux-pr-preview/1133/storybooks/popovers-storybook/)
- [split-view-storybook](https://blackbaud.github.io/skyux-pr-preview/1133/storybooks/split-view-storybook/)
- [storybook](https://blackbaud.github.io/skyux-pr-preview/1133/storybooks/storybook/)
- [tabs-storybook](https://blackbaud.github.io/skyux-pr-preview/1133/storybooks/tabs-storybook/)
- [theme-storybook](https://blackbaud.github.io/skyux-pr-preview/1133/storybooks/theme-storybook/)
- [tiles-storybook](https://blackbaud.github.io/skyux-pr-preview/1133/storybooks/tiles-storybook/)

Apps:

- [code-examples](https://blackbaud.github.io/skyux-pr-preview/1133/code-examples/)
- [dep-graph](https://blackbaud.github.io/skyux-pr-preview/1133/dep-graph/)
- [integration](https://blackbaud.github.io/skyux-pr-preview/1133/integration/)
- [playground](https://blackbaud.github.io/skyux-pr-preview/1133/playground/)

**Review by johnhwhite — APPROVED** [review]

**Pull request merged**